.clippy.toml

```diff
@@ -0,0 +1 @@
+upper-case-acronyms-aggressive = false
```

Comment:
I'd remove this setting and fix the `IOError` name. I will do it next time I touch the code :-P

Reply:
done